CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+- [Pre-built images](#pre-built-images)
+- [Build Requirements](#build-requirements)
+- [Building Executables and Images from Scratch](#building-executables-and-images-from-scratch)
+- [Development](#development)
+- [Local Testing](#local-testing)
+
+## Pre-built images
+
```

Comment:
I would start with something like "you can use already pre-built images for various versions and Tornjak components. For a full list of public images... " 
"Otherwise, you can follow instructions below to build Tornjak images ... "